docs/source/quicktour.mdx

```diff
@@ -308,10 +308,11 @@ The model outputs the final activations in the `logits` attribute. Apply the sof
 >>> import tensorflow as tf
 
 >>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
->>> print(tf.math.round(tf_predictions * 10**4) / 10**4)
-tf.Tensor(
-[[0.0021 0.0018 0.0116 0.2121 0.7725]
- [0.2084 0.1826 0.1969 0.1755  0.2365]], shape=(2, 5), dtype=float32)
+>>> tf_predictions
+<tf.Tensor: shape=(2, 5), dtype=float32, numpy=
+array([[0.00205667, 0.00176607, 0.01154934, 0.21209256, 0.7725354 ],
+	   [0.20841935, 0.1826222 , 0.19692972, 0.17550416, 0.2365246 ]],
+	  dtype=float32)>
```

Comment:
This is super flaky and won't pass on any other machine or at the new TF release. We need to test at a lesser precision or ignore the output.

Reply:
A newbie question, is this block

```
>>> import tensorflow as tf

>>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
>>> tf_predictions
<tf.Tensor: shape=(2, 5), dtype=float32, numpy=
array([[0.00205667, 0.00176607, 0.01154934, 0.21209256, 0.7725354 ],
	   [0.20841935, 0.1826222 , 0.19692972, 0.17550416, 0.2365246 ]],
	  dtype=float32)>
```
independent from other block in the `mdx` file? If so, `tf_outputs` is not defined here ...(?)

Reply:
I tried to fight with it too on a branch with little success. I'd advocate to not lose sleep on it and just ignore the output since we can't get a result that passes both locally for me and on the runner :-)